eng/Version.Details.xml

```diff
@@ -49,331 +53,335 @@
       <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
       <Sha>df4350283ae7ff0c67e08af16c0f00fdbd5e22ac</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Analyzer.Testing" Version="3.0.0-preview.19105.3">
+    <Dependency Name="Microsoft.AspNetCore.Analyzer.Testing" Version="3.0.0-preview.19105.4">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>9750bf67ab22d8d28255c6222be282f4c8e49383</Sha>
+      <Sha>000</Sha>
```

Comment:
Suggest making this (`000`) and the document (`000000`) consistent

Reply:
This will get blown away by the next bot update (https://github.com/aspnet/AspNetCore/pull/7301). I merged this fix as is to unblock the automation that was incorrectly generating PRs to this repo.